msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -910,6 +910,28 @@ Delegated permissions allow the app to access data on behalf of the signed-in us
     |`MicrophoneStream.Read.User`| Allows the app to read user's microphone stream.|
     |`MeetingParticipantReaction.Read.User`| Allows the app to read user's reactions while participating in a meeting.|
 
+## extensions
+
+**Optional** - array
+
+The set of extensions for this app. Used to specify Outlook add-ins for Teams apps that are extended across Microsoft 365, when the app is used in Outlook.
```

Comment:
How about: 

The set of extensions for this app. Used to specify Outlook Add-ins within a unified manifest for simplified distribution and acquisition within the Microsoft 365 ecosystem.

Reply:
Updated in next commit